src/flutter/shell/platform/linux_embedded/window/linuxes_window_x11.cc

```diff
@@ -137,4 +172,36 @@ void LinuxesWindowX11::SetClipboardData(const std::string& data) {
   clipboard_data_ = data;
 }
 
+void LinuxesWindowX11::HandlePointerButtonEvent(xcb_button_t button,
+                                                bool button_pressed, int16_t x,
+                                                int16_t y) {
+  if (binding_handler_delegate_) {
+    FlutterPointerMouseButtons flutter_button;
+    switch (button) {
+      case kButtonLeft:
+        flutter_button = kFlutterPointerButtonMousePrimary;
+        break;
+      case kButtonRight:
+        flutter_button = kFlutterPointerButtonMouseSecondary;
+        break;
+      case kButtonMiddle:
+        flutter_button = kFlutterPointerButtonMouseMiddle;
+        break;
+      case kButtonBack:
+        flutter_button = kFlutterPointerButtonMouseBack;
+        break;
+      case kButtonForward:
+        flutter_button = kFlutterPointerButtonMouseForward;
+        break;
+      default:
+        break;
```

Comment:
In the default case, I think it is better to output the error log and return.

Reply:
I've added the error handling.